index.html

```diff
@@ -435,28 +435,17 @@ <h3>Proof Representations</h3>
           <h4>DataIntegrityProof</h4>
 
           <p>
-The `verificationMethod` property of the proof MUST be a URL.
-Dereferencing the `verificationMethod` MUST result in an object
-containing a `type` property with the value set to
-`Multikey`.
+A proof contains the attributes specified in the
+<a href="https://www.w3.org/TR/vc-data-integrity/#proofs">Proofs section</a>
+of [[VC-DATA-INTEGRITY]] with the following restrictions.
           </p>
-
           <p>
 The `type` property of the proof MUST be `DataIntegrityProof`.
           </p>
           <p>
 The `cryptosuite` property of the proof MUST be `ecdsa-rdfc-2019` or `ecdsa-jcs-2019`.
           </p>
           <p>
-The `created` property of the proof MUST be an [[XMLSCHEMA11-2]]
-formatted date string.
-          </p>
-          <p>
-The `proofPurpose` property of the proof MUST be a string, and MUST
-match the verification relationship expressed by the verification method
-`controller`.
```

Comment:
I think removing this is ok given that it is covered in:

https://w3c.github.io/vc-data-integrity/#retrieve-verification-method

I filed an issue about cleaning that up a bit here: https://github.com/w3c/vc-data-integrity/issues/236

I'll note that the current text calls this section:

"Section 4: Retrieving Cryptographic Material" instead of "Section 4: Retrieve Verification Method".

I'll also note that the signing section here has the same incorrect title (I'm not sure if some respec magic could keep these things up-to-date), but the way it's phrased there also implies one can get `privateKeyBytes` from a verification method, which isn't a thing. I think it's just trying to say one gets the private key bytes associated with the verification method identified by `options.verificationMethod`:

> Let privateKeyBytes be the result of retrieving the private key bytes associated with the options.verificationMethod value as described in the Data Integrity [[VC-DATA-INTEGRITY](https://w3c.github.io/vc-di-ecdsa/#bib-vc-data-integrity)] specification, [Section 4: Retrieving Cryptographic Material](https://www.w3.org/TR/vc-data-integrity/#algorithms).

Referencing the algorithm should probably just be removed then and we could say:

> Let privateKeyBytes be the result of retrieving the private key bytes associated with the verification method identified by the options.verificationMethod value.

But we should probably say: "...privateKeyBytes (or a signing interface enabling the use of the private key bytes) associated with...".

Reply:
Hi @dlongley updated text per your suggestion.